src/webgpu/api/operation/rendering/depth_bias.spec.ts

```diff
@@ -261,3 +339,40 @@ g.test('depth_bias')
   .fn(async t => {
     t.runDepthBiasTest('depth32float', t.params);
   });
+
+g.test('depth_bias_24bit_format')
+  .desc(
+    `
+  Tests that a square with different depth bias values like 'positive', 'negative', 'infinity',
```

Comment:
Infinity is not tested (and can't be tested because WebIDL doesn't allow it)
```suggestion
  Tests that a square with different depth bias values like 'positive', 'negative',
```